app/components/ProfileCard.js

```diff
@@ -43,10 +43,71 @@ class ProfileCard extends Component {
         }).catch(err => console.error('An error occured ', err));
     }
 
+    getRankIcon(rankTier, leaderboardRank) {
+        const rankIcons = {
```

Comment:
can port these into a rank_icon.js file? and export it there?

Reply:
and startIcons as well

Reply:
done